kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/ModemManagerDbusWrapper.java

```diff
@@ -275,4 +287,82 @@ protected void resetHandlersDisable(String deviceId) {
             this.modemHandlers.remove(deviceId);
         }
     }
+
+    protected void failedModemResetTimerEnable(String deviceId, Optional<String> modemManagerDbusPath, int delayMinutes)
```

Comment:
Nitpicking here. This probably is a `Schedule` more than an `Enable`. Can be left as-is.

```suggestion
    protected void scheduleFailedModemResetTimer(String deviceId, Optional<String> modemManagerDbusPath, int delayMinutes)
```

Reply:
Yeah... indeed I chose that names to be consistent with the other Timer/Handler since they are related in some way. Let's me think about it.